### DISCUSSION TITLE
public/robots.txt: Google Search Console shows high fail rate

### DISCUSSION THREAD

**Discussion by MikeEnRegalia** [vercel/next.js#60340] [Help]
### Summary

I'm hosting a hobby project on Vercel (free tier) with a custom domain. I have a robots.txt file in the public folder which Google claims can't be fetched occasionally, with failure rates exceeding 10% on some days.

How can that be? Google also claims that general server availability is problematic. Since I'm on the free tier I cannot investigate the logs too closely, but I'm not aware of any errors the website creates, and the Google Search Console is not showing any 5xx errors.

Any ideas? is this a general problem with Vercel free tier projects?  

![image](https://github.com/vercel/next.js/assets/37114951/b3546a60-66d4-4d5f-bc95-73220f3a715a)

**Comment by don-esteban** [comment]
Could it be that long inactivities cause a sleep mode? Just a wild guess. I never heard about problems with reading static resources on Vercel.

**Comment by rasco** [comment]
Currently experiencing the same issue.
Running NextJS 14.0.4 on Vercel on Pro Tier

**Comment by joaomlneto** [comment]
Same here. Server connectivity is fine — it's probably lambdas timing out. But `robots.txt` is a simple static file.

<img width="922" alt="image" src="https://github.com/vercel/next.js/assets/1539767/13617003-4874-4ce6-a362-062b50d030bd">
